hyperspy/misc/eels/electron_inelastic_mean_free_path.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+
+def _F(beam_energy):
+    return (1 + beam_energy / 1022) / (1 + beam_energy / 511) ** 2
+
+def _theta_E(density, beam_energy):
+    gamma = 1 + beam_energy/511 # keV
+    return 5.5 * density ** 0.3 / (gamma * _F(beam_energy) * beam_energy)
+
+def iMFP(density, beam_energy):
```

Comment:
Shall we add this function and `iMFP_angular_correction` to the API? For example, it would be convenient to be able to do: 

```python
import hyperspy.api as hs
hs.eels.inelastic_mean_free_path(density, beam_energy)
```

Reply:
I suggest we don't in order to stick to HyperSpy's data analysis mission. Once we split HyperSpy we can break the EELS features into 2 packages: a HyperSpy extension for EELS analysis and a EELS package independent from HyperSpy that could have functions like those ones. Does it sound good?

Reply:
I am fine either way.